src/components/vm/disks/diskAdd.tsx

```diff
@@ -738,23 +887,26 @@ const AddDiskModalFooter = ({
     };
 
     const onAddClicked = () => {
+        if (!diskParams.target)
+            return Promise.resolve();
```

Comment:
That should also either become its own commit as a (potential) bug fix, or be replaced with an assert that .target is defined.

Reply:
(Thinking out loud: I don't know why Cockpit makes "target" mandatory, actually... It seems to be guaranteed to be in a disk XML when we get it from libvirt, but I don't think it needs to be in it when we define a new disk...)